apps/evm/src/app/[lang]/(bridge)/utils/stakeData.tsx

```diff
@@ -0,0 +1,170 @@
+import { Flex, P } from '@gobob/ui';
+import { Trans } from '@lingui/macro';
+
+enum Incentives {
+  spice,
+  pell,
+  beckrock,
+  segment,
+  babylon,
+  solv,
+  supply
+}
+
+export type StakingInfo = Record<string, (typeof stakingInfo)[keyof typeof stakingInfo] | undefined>;
```

Comment:
`import { ValueOf } from 'viem';`

```suggestion
export type StakingInfo = Record<string, ValueOf<typeof stakingInfo> | undefined>;
```